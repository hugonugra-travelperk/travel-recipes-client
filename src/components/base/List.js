import styled from "styled-components";

export const ListItem = styled.li`
  margin: 20px 0;
`;
