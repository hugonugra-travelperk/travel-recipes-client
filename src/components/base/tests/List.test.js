import {render, cleanup} from "@testing-library/react"
import "@testing-library/jest-dom";
import {ListItem} from "../List"

afterEach(cleanup);

it("renders", () => {
  const {asFragment} = render(<ListItem>Test text</ListItem>);
  expect(asFragment()).toMatchSnapshot();
});
