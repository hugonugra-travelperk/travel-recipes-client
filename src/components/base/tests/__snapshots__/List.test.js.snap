// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <li
    class="sc-bdvvtL cPxLbc"
  >
    Test text
  </li>
</DocumentFragment>
`;
